apps/web/src/services/landing/components/workspace-landing/landing-group-workspaces/LandingWorkspaceGroupAddUsersModal.vue

```diff
@@ -0,0 +1,305 @@
+                               style-type="secondary"
+                >
+                    <template #default>
+                        <p-select-dropdown
+                            is-filterable
+                            use-fixed-menu-style
+                            class="user-select-dropdown"
+                            style-type="TERTIARY_ICON_BUTTON"
+                            disable-handler
+                            multi-selectable
+                            :menu="userDropdownState.menuList"
+                            :selected.sync="userDropdownState.selectedItems"
+                            :search-text.sync="userDropdownState.searchText"
+                            :loading="userDropdownState.loading"
+                            :placeholder="$t('IAM.WORKSPACE_GROUP.MODAL.USER_DROP_DOWN_PLACEHOLDER')"
+                            @click-show-more="handleClickShowMoreUser"
+                        >
+                            <template #menu-item--format="{ item }">
+                                <div class="user-menu-item">
+                                    <span class="role-user_id">{{ item.user_id }}({{ item.name }})</span>
+                                </div>
+                            </template>
+                        </p-select-dropdown>
+                    </template>
+                </p-field-group>
+            </div>
+            <div v-for="item in userDropdownState.selectedItems"
+                 :key="item.user_id"
+                 class="selected-user-item"
+            >
+                <div class="flex items-center gap-2">
+                    <p-avatar class="menu-icon"
+                              size="md"
+                    />
+                    {{ item?.user_id }}
+                </div>
+                <p-icon-button name="ic_close"
+                               size="md"
+                               @click="handleRemoveUser(item)"
+                />
+            </div>
+        </template>
+        <template #confirm-button>
+            {{ $t('IAM.WORKSPACE_GROUP.MODAL.ADD') }}
+        </template>
+    </p-button-modal>
+</template>
+
+<style lang="postcss" scoped>
+.workspace-group-add-users-modal {
+    .role-select-dropdown {
+        .role-type-icon {
+            @apply rounded-full;
+            width: 1.5rem;
+            height: 1.5rem;
+        }
+
+        .role-menu-item {
+            @apply flex items-center;
+            gap: 0.25rem;
+
+            .role-label {
+                @apply truncate;
+                flex-grow: 1;
+            }
+
+            .role-type {
+                @apply text-label-sm text-gray-400;
+            }
+        }
+    }
+
+    .user-select-dropdown {
+        .user-menu-item {
+            @apply flex items-center justify-between;
+            gap: 0.25rem;
+        }
+
+        :deep(.dropdown-button)::before {
+            content: "Add Users";
+        }
```

Comment:
nitpick: need comment here! 
/* custom design-system component - p-select-dropdown */

+ no need to translate?😮

Reply:
I will remove it.. Thx!